backend/cybnetics/endpoints.py

```diff
@@ -1,8 +1,118 @@
-from flask import Flask, jsonify
+import json
+import ast
+import jwt
+import datetime
+
+from flask import Flask, jsonify, request, session
 
 app = Flask(__name__)
 app.config.from_pyfile('./config.py')
 
-@app.route('/test', methods=['GET'])
-def test():
-    return jsonify({'hello': 'world'})
+
+db = app.config['DATABASE']
+
+# This is just a sanity check from myself
+@app.route("/")
+def get_initial_response():
```

Comment:
its good to test things, but i think we can remove this function if its never going to get used